utils/podcast.py

```diff
@@ -0,0 +1,189 @@
+import spotipy
+from spotipy.oauth2 import SpotifyOAuth
+import random
+import logging
+
+
+# implement threading for the podcast uploading 
+logger = logging.getLogger("root")
+
+TOPIC_LIST = [
```

Comment:
maybe this data could be loaded from a text file or even just another python file. not a big deal but makes the file a little overwhelming